langchain-core/src/runnables/base.ts

```diff
@@ -550,6 +551,29 @@ export abstract class Runnable<
     );
   }
 
+  getGraph(config?: RunnableConfig): Graph {
+    const graph = new Graph();
+
+    // TODO: what should the inputNode be?
+    const inputNode = graph.addNode({
+      type: "unknown",
+      config,
+    });
```

Comment:
@jacoblee93 Here's where I'm a little bit lost,

I'm trying to find a way to put the schema of the input node of this runnable. I haven't been able to find it yet.

This should be similar to [get_input_schema](https://github.com/langchain-ai/langchain/blob/a99eb3abf4db7ce98fb271df93c23867583782b8/libs/core/langchain_core/runnables/base.py#L393) in Python's core lib. 

I think for graphing purposes, we might be able to replace this with a placeholder and maybe implement this later when we're going to use Zod.